backend/controllers/followups.ts

```diff
@@ -217,6 +217,10 @@ export async function logSurveyLinkClick(req: Request, res: Response) {
     const redirectUrl = await getRedirectUrl(req)
     res.redirect(redirectUrl)
   } catch (error) {
+    console.error(
+      "Erreur lors de la mise à jour du sondage dans le followup",
```

Comment:
Jusqu'à présent on a les erreurs en anglais
```suggestion
      "Error: survey update in followup",
```